frontend/app/src/store/session/index.ts

```diff
@@ -102,6 +104,7 @@ export const useSessionStore = defineStore('session', () => {
       fetchNetValue()
     ]);
     await refreshPrices();
+    await refreshBlockchainBalances();
```

Comment:
If you follow `fetch` you can see that it already does get accounts/blockchain balances. By adding refresh you will force one more refresh to happen really close to the previous one.

Reply:
I'll remove that line